<!--  Create a launch file to start the whole simulation -->

<launch>
    <!-- Launch node inside assignment2_2024 -->
    <include file="$(find assignment_2_2024)/launch/assignment1.launch" />

    <!-- Launch node inside assignment2_ros1 -->
    <node pkg="assignment2_ros1" type="action_client_node.py" name="action_client_node" output="screen"/>
    <!--  <node pkg="assignment2_ros1" type="service_client_node" name="service_client_node" /> -->
</launch>